### PR TITLE
DOC-3356 Link Fixes CE 4.4 (Part 2)

### PR DESCRIPTION
**Pull request by AaronForinton**
Covers everything from the Notary section down.

### CODE REVIEW

File: content/en/platform/corda/4.4/enterprise/app-upgrade-notes.md [hunk 1]
```diff
@@ -780,10 +780,9 @@ to be governed by a contract that is either:
 * The outer class of the state class, if the state is an inner class of a contract. This is a common design pattern.
 * Annotated with `@BelongsToContract` which specifies the contract class explicitly.
 
-Learn more by reading [Contract/State Agreement](../../../../../en/platform/corda/4.4/enterprise/cordapps/api-contract-constraints.html#contractstate-agreement). If an app targets Corda 3 or lower (i.e. does not specify a target version),
+Learn more by reading [Contract/State Agreement](../../../../../en/platform/corda/4.4/enterprise/cordapps/api-contract-constraints.html#contract-state-agreement). If an app targets Corda 3 or lower (i.e. does not specify a target version),
```
Comment (michelemarsden) [inline content/en/platform/corda/4.4/enterprise/app-upgrade-notes.md:783]: This one was originally correct. The header is Contract/State Agreement, so no space between Contract and State in the link text. 

```suggestion
Learn more by reading [Contract/State Agreement](../../../../../en/platform/corda/4.4/enterprise/cordapps/api-contract-constraints.html#contractstate-agreement). If an app targets Corda 3 or lower (i.e. does not specify a target version),
```

File: content/en/platform/corda/4.4/enterprise/financial-model.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ These provide a common language for states and contracts.
 
 ## Amount
 
-The [Amount](api/kotlin/corda/net.corda.core.contracts/-amount/index.html) class is used to represent an amount of
+The [Amount](../../../../../../archived-docs/corda-enterprise/api-ref/3.3/html/api/kotlin/corda/net.corda.core.contracts/-amount) class is used to represent an amount of
```
Comment (michelemarsden) [inline content/en/platform/corda/4.4/enterprise/financial-model.md:29]: This link is broken still.

File: content/en/platform/corda/4.4/enterprise/network/network-map.md [hunk 3]
```diff
@@ -102,7 +102,7 @@ the network, along with the network parameters file and identity certificates. G
 online at once - an offline node that isn’t being interacted with doesn’t impact the network in any way. So a test
 cluster generated like this can be sized for the maximum size you may need, and then scaled up and down as necessary.
 
-More information can be found in [Network Bootstrapper](../../../../../../en/platform/corda/4.4/enterprise/network-bootstrapper.md).
+More information can be found in network-bootstrapper.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.4/enterprise/network/network-map.md:105]: Should this still be a link?

File: content/en/platform/corda/4.4/enterprise/network/network-map.md [hunk 4]
```diff
@@ -180,7 +180,7 @@ Any CorDapp JAR that offers contracts and states in any of these packages must b
 This ensures that when a node encounters an owned contract it can uniquely identify it and knows that all other nodes can do the same.
 Encountering an owned contract in a JAR that is not signed by the rightful owner is most likely a sign of malicious behaviour, and should be reported.
 The transaction verification logic will throw an exception when this happens.
-Read more about Package ownership in the [Package namespace ownership](../node/deploy/env-dev.md#package-namespace-ownership) section.
+Read more about *Package ownership* here “[Package namespace ownership](../../../../../../en/platform/corda/4.4/enterprise/node/deploy/env-dev.md#package-namespace-ownership)”.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.4/enterprise/network/network-map.md:183]: Not sure on the italics of things in quotes on this one...

```suggestion
Read more about [package namespace ownership](../../../../../../en/platform/corda/4.4/enterprise/node/deploy/env-dev.md#package-namespace-ownership)”.
```

File: content/en/platform/corda/4.4/enterprise/node-explorer.md [hunk 5]
```diff
@@ -142,7 +142,7 @@ To see a geographical view of your network, click **Network** in the left-hand s
 
 {{< note >}}
 
-The geographical location of each node is based on the [`locality` property of the node's registered CordaX500 name](https://api.corda.net/api/corda-os/4.4/html/api/kotlin/corda/net.corda.core.identity/-corda-x500-name/index.html)
+The geographical location of each node is based on the [`locality` property of the node's registered CordaX500 name](../../../../../../archived-docs/corda-enterprise/api-ref/3.3/html/api/kotlin/corda/net.corda.core.identity/-corda-x500-name/index.html)
```
Comment (michelemarsden) [inline content/en/platform/corda/4.4/enterprise/node-explorer.md:145]: This link is broken and you'll need to use html link as backticks don't render properly.

File: content/en/platform/corda/4.4/enterprise/notary/ha-notary-service-setup.md [hunk 6]
```diff
@@ -105,7 +105,7 @@ entry. If configured to use an HSM, the generated keys are stored in the HSM and
 
 In order for network participants to use the new HA notary the notary service must be present in the network parameters. This involves
 configuring and setting the initial network parameters (if setting up a new network), or modifying the existing parameters and performing a
-flag day (if using an existing network). Please refer to the [CENM documentation](../../../cenm/1.2.html) for more information on this
+flag day (if using an existing network). Please refer to the [CENM documentation](../../../../../../en/platform/corda/1.2/cenm/_index.md) for more information on this
```
Comment (michelemarsden) [inline content/en/platform/corda/4.4/enterprise/notary/ha-notary-service-setup.md:108]: This link is broken.

File: content/en/platform/corda/4.4/enterprise/notary/hsm-support.md [hunk 7]
```diff
@@ -43,21 +43,21 @@ See the *enterpriseConfiguration* section of the corda-configuration-file doc fo
 When the private keys are stored in a HSM, only the certificates are stored in the keystore file.
 
 The associated certificates for the distributed notary identity and node certificate authority are issued by the Identity Manager on the
-network. See the [Corda OS documentation](https://docs.corda.net/docs/corda-os/index.html) for more information. The legal identity certificate is issued by the node certificate authority.
+network. See the [Corda OS documentation](../../../../../../en/platform/corda/4.4/open-source/_index.md) for more information. The legal identity certificate is issued by the node certificate authority.
```
Comment (michelemarsden) [inline content/en/platform/corda/4.4/enterprise/notary/hsm-support.md:46]: This link is broken.

**Review by michelemarsden — COMMENTED** [review]
Please see my comments.

**Review by AnnaK-R3 — APPROVED** [review]

**Pull request merged**